### PR TITLE
New parser for xml data sources

### PR DESCRIPTION
**Pull request by Iva-khar**
New parser for xml data sources

### CODE REVIEW

File: data_ocean/converter.py [hunk 1]
```diff
@@ -222,7 +223,22 @@ def process(self):
         except:
             None
         print('All the records have been rewritten.')
-
+    
+    def process_full(self): # It's temporary method name, in the future this 'process' will be one       
+        i = 0
+        records = etree.Element('RECORDS')
+        for _, elem in etree.iterparse(self.LOCAL_FOLDER + self.LOCAL_FILE_NAME, tag = 'SUBJECT'):
```
Comment (Anntroy) [inline data_ocean/converter.py:230]: "_," what does it mean?

File: data_ocean/converter.py [hunk 2]
```diff
@@ -222,7 +223,22 @@ def process(self):
         except:
             None
         print('All the records have been rewritten.')
-
+    
+    def process_full(self): # It's temporary method name, in the future this 'process' will be one       
+        i = 0
+        records = etree.Element('RECORDS')
+        for _, elem in etree.iterparse(self.LOCAL_FOLDER + self.LOCAL_FILE_NAME, tag = 'SUBJECT'):           
+            if len(records) < self.CHUNK_SIZE:
```
Comment (Anntroy) [inline data_ocean/converter.py:231]: Where do you define CHUNK_SIZE?

Comment (Iva-khar) [reply data_ocean/converter.py:231]: It will in services

File: data_ocean/converter.py (hunk 1 above)
Comment (Iva-khar) [reply data_ocean/converter.py:230]: some required but no realy need 
variable

**Pull request merged**